low-level/files/opt/seagate/sspl/setup/sspl_config.py

```diff
@@ -0,0 +1,330 @@
+            os.makedirs(self.LOGROTATE_DIR)
+
+        shutil.copy2(f'{self.DIR_NAME}/low-level/files/etc/logrotate.d/iem_messages', self.IEM_LOGROTATE_CONF)
+        shutil.copy2(f'{self.DIR_NAME}/low-level/files/etc/logrotate.d/sspl_logs', self.SSPL_LOGROTATE_CONF)
+        
+        # This rsyslog restart will happen after successful updation of rsyslog
+        # conf file and before sspl starts. If at all this will be removed from
+        # here, there will be a chance that SSPL intial logs will not be present in
+        # "/var/log/<product>/sspl/sspl.log" file. So, initial logs needs to be collected from
+        # "/var/log/messages"
+        try :
+            Utility.systemctl_service_action('rsyslog.service', action='restart')
+        except OSError as error:
+            sys.exit("Error: Failed to restart rsyslog.service with an error : \n", error)
+
+        # For node replacement scenario consul will not be running on the new node. But,
+        # there will be two instance of consul running on healthy node. When new node is configured
+        # consul will be brought back on it. We are using VIP to connect to consul. So, if consul
+        # is not running on new node, we dont need to error out.
+        # If consul is not running, exit
+        # Onward LDR_R2, consul will be abstracted out and it won't
+        # exit as hard dependeny of SSPL
+
+        if consts.PRODUCT_NAME == 'LDR_R1':
+            if not os.path.exists(consts.REPLACEMENT_NODE_ENV_VAR_FILE):
+                CONSUL_PS = None
+                for proc in psutil.process_iter():
+                    if 'consul' in proc.name():
+                        CONSUL_PS = proc.pid
+                if not CONSUL_PS:
+                    sys.stderr.write("Consul is not running, exiting..")
+                    sys.exit(1)
+
+        # Get the types of server and storage we are currently running on and
+        # enable/disable sensor groups in the conf file accordingly.
+        if read_provisioner_config:
+            # subprocess.call(['python3', f'{self._script_dir}/conf_based_sensors_enable'])   
+            update_sensor_info()    
+            
+    def get_rabbitmq_cluster_nodes(self):
+        pout = None
+        if self.rabbitmq_major_release == '3' and self.rabbitmq_maintenance_release == '8':
+            rabbitmq_cluster_status = Utility.send_command('/usr/sbin/rabbitmqctl cluster_status --formatter json')
+            rabbitmq_cluster_status = json.loads(rabbitmq_cluster_status)
+            running_nodes = rabbitmq_cluster_status['running_nodes']
+            for i, node in enumerate(running_nodes):
+                running_nodes[i] = node.replace('rabbit@', '')
+            pout = " ".join(running_nodes)
+        elif self.rabbitmq_version == '3.3.5':
+            out = Utility.send_command("rabbitmqctl cluster_status | grep running_nodes | cut -d '[' -f2 | cut -d ']' -f1 | sed 's/rabbit@//g' | sed 's/,/, /g'")
```

Comment:
After "rabbitmqctl cluster_status", I feel that the formatting should be done in python, but how should I get the output for rabbitmq version 3.3.5? I will need to check in what format it exactly gives the output to convert it in python